Cubical/Categories/Displayed/Constructions/IsomorphismMore.agda

```diff
@@ -0,0 +1,65 @@
+{-# OPTIONS --safe #-}
+-- TODO this gets moved to Categories.Isomorphism
```

Comment:
This doesn't have anything to do with displayed categories, move it to Categories.Isomorphism.More or something plz